c_src/membrane_audio_mix_plugin/stream_format_audio_raw.c

```diff
@@ -97,6 +101,9 @@ int64_t caps_audio_raw_sample_min(CapsAudioRaw *caps) {
 /**
  * Returns byte size for given format.
  */
-uint8_t caps_audio_raw_sample_byte_size(CapsAudioRaw *caps) {
-  return (uint8_t)((caps->sample_format & MEMBRANE_SAMPLE_FORMAT_SIZE) / 8);
+uint8_t
+stream_format_audio_raw_sample_byte_size(StreamFormatAudioRaw *stream_format) {
+  return (
+      uint8_t)((stream_format->sample_format & MEMBRANE_SAMPLE_FORMAT_SIZE) /
+               8);
```

Comment:
Is this formatted with `clang-format`? Looks quite weird

Reply:
It is